notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb

```diff
@@ -1034,38 +986,38 @@
         "id": "explanation_spec"
       },
       "source": [
-        "### Explanation Specification\n",
+        "### Explanation specification\n",
         "\n",
-        "To get explanations when doing a prediction, you must enable the explanation capability and set corresponding settings when you upload your custom model to an Vertex `Model` resource. These settings are referred to as the explanation metadata, which consists of:\n",
+        "To get explanations when doing a prediction, you must enable the explanation capability and set corresponding settings when you upload your custom model to an Vertex AI model resource. These settings are referred to as the explanation metadata, which consists of:\n",
         "\n",
         "- `parameters`: This is the specification for the explainability algorithm to use for explanations on your model. You can choose between:\n",
-        "  - Shapley - *Note*, not recommended for image data -- can be very long running\n",
+        "  - Shapley(**Note**: can be very long running and not recommended for image data)\n",
```

Comment:
suggeste edit:

- "Not recommended for image data since it can be very long running."